### PR TITLE
custom user attribute handler docu

### PR DESCRIPTION
**Pull request by jona-samuel**
This is the first version of the docu for the custom user attribute event handler. Feedback is very welcome.

### CODE REVIEW

**Comment by cornelinux** [comment]
@jona-samuel Please rebase this PR on the `branch-3.7`. Get in touch if unclear how to do this locally.

File: doc/eventhandler/customuserattributehandler.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+if a user failed to log-in to many time, the eventhandler can automatically set the custom attribute
+"this user has log-in problems".
+
+delete_custom_user_attributes
+................
+
+With this action you can automatically delete custom_user_attributes. For example you can give
+a token for a new user and set the custom user attribute "new user maybe needs help by the
+first login" and if the user loges-in for the first time the attribute can be automatically deleted.
+
```
Comment (cornelinux) [inline doc/eventhandler/customuserattributehandler.rst:29]: My Suggestion:

---
This action deletes a custom user attribute on a certain event.

The action takes the options `attrkey` and `user` as described above.

For example you could set a custom user attribute on the token enrollment process, that indicates, that this is a new user and set a custom user attribute to "new user maybe needs help by the first login".
On the first successful login this custom user attribute could be automatically deleted using this event handler.

**Review by cornelinux — CHANGES_REQUESTED** [review]
Hi @jona-samuel 
Please see my suggestions.

Also ensure:

* ~~set the link to this documentation in the index.rst~~
* rebase to 3.7
* ~~check if tags are allowed or not.~~

File: doc/eventhandler/customuserattributehandler.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Possible Actions
+~~~~~~~~~~~~~~~~
+
+set_custom_user_attributes
+................
+
+You can set a custom_user_attribute automatically if a certain event is triggered. For example
+if a user failed to log-in to many time, the eventhandler can automatically set the custom attribute
+"this user has log-in problems".
```
[the remark this answers is not in the document]
Comment (jona-samuel) [reply doc/eventhandler/customuserattributehandler.rst:21]: this is correct

File: doc/eventhandler/customuserattributehandler.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+Possible Actions
+~~~~~~~~~~~~~~~~
+
+set_custom_user_attributes
+..........................
+
+This action sets a custom user attribute on a certain event.
+
+The action takes the options attrkey, attrvalue and user.
```
Comment (cornelinux) [inline doc/eventhandler/customuserattributehandler.rst:26]: Please mark the names of the options "attrkey" and "attrvalue", this way it is easier to read.

You can do this with the backticks "`" before and after the word.

Also check for this in the remaining text.

**Review by cornelinux — CHANGES_REQUESTED** [review]
Thanks a lot.
This is better readable. However, my last request is, that you format certain words (e.g. with backticks), that indicate as key words, this also helps to logically understand the text better.

**Comment by jona-samuel** [comment]
Replaced with new PR

**Pull request closed without merging**